components/OurPartners/OurPartners.module.css

```diff
@@ -0,0 +1,27 @@
+.container {
+  font-family: Nunito;
+}
+
+.header {
+  font-size: 3.75rem;
+  transform: translate(-5.25rem);
```

Comment:
Could you explain what the transform is used to do in this file? The title and top paragraphs look a little off-center.

Reply:
I updated the sizes for both the page heading and the card titles at smaller sizes.

Reply:
The transform moves the header and the subheader to line up with the left edge of the blue tabs on each card (as the actual left margins are the left edge of the cards and the blue tabs are shifted to the left of the margin)
The education pages have the h1 and subhead left aligned with the margins. For some reason in the Figma the "Our Team" and "Our Mission and Goals" pages were designed to line up with the left edge of the blue tabs, so that's what I did.
If you'd like, I could easily re-align them to the left edge of the white cards like the education pages. To my eye, they look too close to the left edge and it makes the blue tabs have less impact.